CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Here is a template for new release sections
 - KPI processing function `E3.add_total_consumption_from_provider_electricity_equivalent()` incl. pytest (#726)
 - Subsection `Minimal degree of autonomy constraint` for RTD, added parameter in parameter csv  (#730, #726)
 - Minimal degree of autonomy constraint including pytest and benchmark test (#726)
+- Add test for json benchmark from EPA (#781)
```

Comment:
```suggestion
- Add benchmark test for json benchmark file from EPA (EPA-MVS compatability) (#781)
```
Can you add a mention of the benchmark test [here](https://multi-vector-simulator.readthedocs.io/en/latest/Model_Assumptions.html#benchmark-tests)?